### PR TITLE
fixes for configuration script

### PR DESCRIPTION
**Pull request by tishma**
also fixed some JSLint complaints

### CODE REVIEW

File: service-configure.js [hunk 1]
```diff
@@ -179,6 +180,15 @@
         var questions = [
             {
                 type: 'input',
+                name: 'testfairyConnectEndpoint',
+                message: 'What is your TestFairy Connect endpoint (do not change unless you have custom testfairy domain)?',
```
Comment (gmegidish) [inline service-configure.js]: All developers now have a custom domain (you wrote that code ;)). Can you please change to "What is your TestFairy Connect endpoint (eg, https://acme.testfairy.com/connect)?

**Review by gmegidish — APPROVED** [review]
Looks good! let's merge :)

**Comment by tishma** [comment]
Well, besides merge, we own the world releases of testfairy-connect and
Docker container

**Comment by tishma** [comment]
owe

**Pull request closed without merging**